cfgov/v1/util/util.py

```diff
@@ -143,3 +144,14 @@ def all_valid_destinations_for_request(request):
                             valid_destination_for_request(request, pair[1])]
 
     return valid_destinations
+
+
+def get_streamfields(page):
+    """
+    Retrieves the stream values on a page from it's Streamfield
```

Comment:
Nitpicky grammar, should be "its".

More seriously, what if you have multiple Streamfields on a page, with identically-named fields? I don't know if this exists right now in our page classes but it doesn't seem impossible, e.g.

```py
class PageWithMultipleStreamfields(Page):
    section1 = StreamField([
        ('introduction', molecules.TextIntroduction()),
        ('content', blocks.RichTextBlock()),
    ])
    section2 = StreamField([
        ('introduction', molecules.TextIntroduction()),
        ('content', blocks.RichTextBlock()),
    ])
```
This method would combine multiple blocks into a single dictionary value.